docs/genai/howto/install.md

```diff
@@ -23,61 +27,120 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
 
 Ensure that the `CUDA_PATH` environment variable is set to the location of your CUDA installation.
 
+Versions later than ONNX Runtime release bundle CUDA 12 by default. CUDA 11 is also supported in a separate package. Instructions are given below.
+
 ## Python packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these set of packages (CPU, DirectML, CUDA) should be installed in your environment.
 
 ### CPU
 
+#### Intel CPU
+
 ```bash
-pip install numpy
+pip install onnxruntime
```

Comment:
We need at least onnxruntime==1.19.0 for genai to work, because previous version don't have the libraries we need.

Reply:
Reverted Python instructions for now. We will wait for ORT release